blog/_posts/2017-09-18-story-of-an-issue.md

```diff
@@ -0,0 +1,100 @@
+---
+title: A story of an issue  
+layout: post
+created_at: Mon Sept 18 2017
+permalink: blog/2017-09-18-story-of-an-issue
+current: blog
+author: Neta and Rete
+twitter: ruby_bears
+categories:
+- student-posts
+---
+
+# A story of an issue
```

Comment:
This heading 'A story of an issue' looks bit odd as we already have same blog title. And there is no text between the title and a heading. Can you just remove this heading?